test/test_helper.exs

```diff
@@ -19,6 +22,10 @@ defmodule TestHarness do
     end
   end
 
+  defp check_results(results) do
+    Enum.each results, &(assert &1 == :passed)
```

Comment:
Done! Curious, what's your style on this? I tend to like omitting parens on multiline calls that have anonymous functions.

```elixir
Enum.each list, fn item ->
  # something
  # else
end
```

For single-line, I don't have any clear preference. In this case, I think I opted to leave them out to avoid the double parens at the end.

What's your metric?